cockatrice/src/dlg_settings.cpp

```diff
@@ -805,6 +805,7 @@ MessagesSettingsPage::MessagesSettingsPage()
     connect(&invertHighlightForeground, SIGNAL(stateChanged(int)), this, SLOT(updateTextHighlightColor(int)));
 
     mentionColor = new QLineEdit();
+    mentionColor->setPlaceholderText(tr("Enter HEX code"));
```

Comment:
```suggestion
    mentionColor->setPlaceholderText(tr("Enter hex code"));
```
hex is short for hexadecimal

Reply:
I've seen both.
When you search "color picker" in Google to access their tool, it uses HEX for example.

Reply:
Obsolete with #4535